orbit/pkg/update/notifications_test.go

```diff
@@ -317,11 +317,14 @@ func TestWindowsMDMEnrollmentPrevented(t *testing.T) {
 
 			started := make(chan struct{})
 			go func() {
-				close(started)
+				go func() {
+					// the first call will block in enroll/unenroll func
+					cfg, err := fetcher.GetConfig()
+					assertResult(cfg, err)
+				}()
 
-				// the first call will block in enroll/unenroll func
-				cfg, err := fetcher.GetConfig()
-				assertResult(cfg, err)
+				time.Sleep(100 * time.Millisecond)
+				close(started)
```

Comment:
Wouldn't it be simpler to just `time.Sleep(100ms)` in the main function instead, and just launch one goroutine? If I read this correctly, it would be equivalent (i.e. all that the first goroutine does is wait 100ms and signal on the channel, which then resumes the main function).

Reply:
@mna sounds good! I was hesitant to do that because it felt weird to add a sleep after you waited for the "start" signal.

I was also thinking: why have a `started` channel if we have to add a sleep outside of the function anyways?

I pushed changes to move the sleep out, let me know if that's what you had in mind

Reply:
@roperzh sorry, I meant to get rid of the `started` channel too, since it did nothing but confirm that the sleep was done (which is now unnecessary, we just do the sleep and resume). Unless of course the channel is used later on, but even then, I don't think it's useful in this particular place.

Reply:
@mna ahh, thank you very much and sorry I didn't get that. Done!